src/baldr/graphtile.cc

```diff
@@ -80,44 +80,47 @@ GraphTile::GraphTile(const std::string& tile_dir, const GraphId& graphid) : head
       // read the compressed file into memory
       std::vector<char> compressed(std::istreambuf_iterator<char>(file),
                                    (std::istreambuf_iterator<char>()));
+      // try to decompress it
+      DecompressTile(graphid, compressed);
+    }
+  }
+}
 
-      // for setting where to read compressed data from
```

Comment:
i moved this into its own function so we can call it from multiple code paths, one such code path is seen above